aiohttp/cookiejar.py

```diff
@@ -92,6 +92,13 @@ def __init__(
         except OverflowError:
             self._max_time = self.MAX_32BIT_TIME
 
+    def clone(self) -> AbstractCookieJar:
+        return CookieJar(
```

Comment:
Should probably use `self.__class__` or similar, so it works as expected for subclasses.